### DISCUSSION TITLE
Account still Paused even after new period started

### DISCUSSION THREAD

**Discussion by RyanBaig** [vercel/community#5803] [Account Issue] [answered]
### Question

So basically, my Vercel account was paused because I passed the usage limits on the vercel serverless functions on **Jan11, 2024** *(30 days ago)*
This is the email:
![image](https://github.com/vercel/community/assets/86055075/4ddbaceb-52bb-41ac-97fe-f60e9dbe87c6)
so I searched on Google when it will be unpaused, and on the Hobby Plan (My Plan), it said that the usage limits will reset after 30 days. so 30 days after Jan11 is Feb12, Today is Feb13, the usage limits have reset, but my account is still paused. Any Help please?
![image](https://github.com/vercel/community/assets/86055075/6cc2adcd-a975-4740-a3c2-eab1efc9ecec)

**Comment by amyegan** [comment, marked as answer]
Hi @RyanBaig. Please [submit a case](https://vercel.com/help) to the Customer Success Team with `Usage Enquiry` selected as the Problem Area. That will get you in contact with the team that can review your usage and provide a solution.

[vercel.com/help](https://vercel.com/help)

**Reply by RyanBaig** [reply]
I created a case and my account is unpaused. Thank you for the support :smile:
